app/helpers/guides_helper.rb

```diff
@@ -0,0 +1,9 @@
+module GuidesHelper
+  def book_an_appointment_link?
```

Comment:
@andrewgarner be good to get your thoughts on this... not sure what pattern we follow for helpers